### PR TITLE
WIP: Core long layout

### PR DESCRIPTION
**Pull request by deadlight**
## Done

Change of layout for the /core page

### CODE REVIEW

File: templates/core/index.html [hunk 1]
```diff
@@ -1,199 +1,581 @@
...
 
 {% block content %}
-<div class="p-strip--light is-deep is-bordered">
-  <div class="row">
-    <div class="col-6">
-      <h1>Ubuntu Core</h1>
+<section class="p-strip--image p-banner-core-hero is-deep is-dark">
+  <div class="row u-equal-height">
+    <div class="col-7">
+      <h1>Make secure IoT devices with Ubuntu Core</h1>
```
Comment (Greggless) [inline templates/core/index.html]:
```suggestion
      <h1>Build secure IoT devices with Ubuntu Core</h1>
```

File: templates/core/index.html [hunk 2]
```diff
@@ -1,199 +1,581 @@
...
-      <h4>Secure by default</h4>
-      <p>Automatic updates ensure that critical security issues are addressed in the field, even if a device is unattended.</p>
+    <hr class="p-separator" />
+  </div>
+
+  <div class="row">
+    <div class="col-6">
+      <h4>10 years security updates, by Canonical</h4>
+      <p class='p-heading--five'>That's a decade of sleeping soundly.</p>
+      <p>Ubuntu Core 18 gets 10 years of Canonical maintenance from Ubuntu 18.04 LTS. Your smallest devices are now just as well covered as your servers.</p>
```
Comment (Greggless) [inline templates/core/index.html]:
```suggestion
      <p>Ubuntu Core 18 gets 10 years of Canonical maintenance from Ubuntu 18.04 LTS. Your smallest devices are now as secure as your servers.</p>
```

File: templates/core/index.html [hunk 3]
```diff
@@ -1,199 +1,581 @@
...
+</section>
+
+<section class='p-strip'>
   <div class="row">
-    <div class="col-12  u-align--center">
-      <img src="{{ ASSET_SERVER_URL }}79334500-Ubuntu_Classic_vs_Core_Desktop.svg" alt="" />
+    <div class="col-6">
+      <h4>Manufacturer’s update control</h4>
+      <p class='p-heading--five'>Decide which updates go to your devices.</p>
+      <p>As a device manufacturer or a snap publisher, you decide which updates are certified and delivered to your devices. Digital signatures certify compatibility for precise update control.</p>
```
Comment (Greggless) [inline templates/core/index.html]:
```suggestion
      <p>As a device manufacturer or a snap publisher, you decide which updates are signed, certified and delivered to your devices.</p>
```

File: templates/core/index.html [hunk 4]
```diff
@@ -1,199 +1,581 @@
...
+      <img src="{{ ASSET_SERVER_URL }}08af835e-Snap+Store+shopping+icon.svg" width="200" alt="">
     </div>
-    <div class="col-6 u-align--center">
-      <img src="{{ ASSET_SERVER_URL }}c6a45f9c-Ubuntu_Classic_vs_Core_-mobile2.svg" alt="" width="260" style="margin-top: 1rem;"/>
+    <div class="col-6">
+      <h4>App store included</h4>
+      <p class='p-heading--five'>Your own app store. Your own software portfolio.</p>
+      <p>Every device built with Ubuntu Core has a secure app store. Use the global
+store for access to all the standard apps, or curate your own collection.</p>
+      <p><a class="p-button--neutral is-inline" href="/internet-of-things/appstore" style="margin-left: 0;"><span class="p-link--external">See what an app store can do for you</span></a></p>
```
Comment (Greggless) [inline templates/core/index.html]:
```suggestion
      <p><a class="p-button--neutral is-inline" href="/internet-of-things/appstore" style="margin-left: 0;"><span class="p-link--external">See what an app store can do for your things</span></a></p>
```

File: templates/core/index.html [hunk 5]
```diff
@@ -1,199 +1,581 @@
...
+  </div>
+
+  <div class="row u-equal-height">
     <div class="col-6">
-      <h2>The smallest Ubuntu</h2>
-      <p>Ubuntu Core is smaller than competing &ldquo;micro&rdquo; container OS offerings. It is small because it&rsquo;s really just a base filesystem. Apps are delivered as snaps, alongside a free choice of container runtimes and coordination systems. And because it&rsquo;s got a smaller attack surface, it&rsquo;s much more secure.</p>
+      <h4>Enterprise management</h4>
+      <p class='p-heading--five'>Absolute control over every device in the building.</p>
+      <p>Enterprises gain complete audit and control over every piece of software on every single device on the network. Regardless of manufacturer.</p>
+      <p>You know exactly which kernel version and which OS version is running. You decide when updates happen. You decide which versions, too.</p>
```
Comment (Greggless) [inline templates/core/index.html]:
```suggestion
      <p>Know exactly which kernel version and which OS version is running. You decide when updates happen. You decide which versions, too.</p>
```

**Comment by Greggless** [comment]
Closing as work continued in https://github.com/canonical-websites/www.ubuntu.com/pull/4509

**Comment by anthonydillon** [comment]
@Greggless thanks

**Pull request closed without merging**